eng/templates/stages/release.yml

```diff
@@ -0,0 +1,297 @@
+      - name: SourceBuildArtifactLeg
+        value: CentOSStream8_Offline_x64_Artifacts
+      - name: SourceBuiltArtifactsFileName
+        value: Private.SourceBuilt.Artifacts.8.0.100.centos.8-x64.tar.gz
+      - name: SdkArtifactFileName
+        value: dotnet-sdk-*.tar.gz
+
+    steps:
+    - task: DownloadPipelineArtifact@2
+      name: DownloadSourceBuiltArtifactsStep
+      displayName: Download Source-Built Artifacts
+      inputs:
+        source: specific
+        project: $(AZDO_PROJECT)
+        pipeline: $(officialBuildPipelineId)
+        runVersion: specific
+        runId: $(officialBuildRunId)
+        artifact: $(SourceBuildArtifactLeg)
+        patterns: $(SourceBuiltArtifactsFileName)
+
+    - task: DownloadPipelineArtifact@2
+      name: DownloadSourceBuiltSDKStep
+      displayName: Download Source-Built SDK
+      inputs:
+        source: specific
+        project: $(AZDO_PROJECT)
+        pipeline: $(officialBuildPipelineId)
+        runVersion: specific
+        runId: $(officialBuildRunId)
+        artifact: $(SourceBuildArtifactLeg)
+        patterns: $(SdkArtifactFileName)
+
+    - template: ../steps/upload-to-blob-storage.yml
+      parameters:
+        file: $(PIPELINE.WORKSPACE)/$(SourceBuiltArtifactsFileName)
+        accountName: $(storageAccountName)
+        containerName: $(blobContainerName)
+        uploadPath: $(artifactsUploadBaseFilePath)
+        azureStorageKey: $(dotnetcli-storage-key)
+
+    - template: ../steps/upload-to-blob-storage.yml
+      parameters:
+        file: $(PIPELINE.WORKSPACE)/$(SdkArtifactFileName)
+        accountName: $(storageAccountName)
+        containerName: $(blobContainerName)
+        uploadPath: $(sdkUploadBaseFilePath)
+        azureStorageKey: $(dotnetcli-storage-key)
+
+    - script: |
+        set -euo pipefail
+
+        export RELEASE_DATE=$(date +"%B %Y") # e.g. "March 2022"
+        export RUNTIME_VERSION="$(runtimeVersion)"
+        export SDK_VERSION="$(sdkVersion)"
+
+        template="$(envsubst < source-build-release-pr.md)"
+        # Get the line in the template that is prefixed with "Title:" and remove the prefix
+        title=$(echo "$template" | grep "^Title:" | cut -d " " -f2-)
+        # Get the inverse of the above selection
+        body=$(echo "$template" | grep -v "^Title:")
+
+        cd $(Agent.WorkFolder)/
+
+        echo "TargetRepo: $(TargetRepo)"
+        echo "ForkRepo: $(ForkRepo)"
+        echo "SdkVersion: $(sdkVersion)"
+        echo "title: ${title}"
+        echo "body: ${body}"
+
+        if [ ${{ parameters.isDryRun }} = True ]; then
+          echo "Doing a dry run, not submitting PR."
+        else
+          echo "Submitting PR"
+          ./submit-source-build-release-pr.sh \
+            --setupGitAuth \
+            --targetRepo $(TargetRepo) \
+            --forkRepo $(ForkRepo) \
+            --sdkVersion $(sdkVersion) \
+            --title "${title}" \
+            --body "${body}"
+        fi
```

Comment:
Are the paths to version.props and global.json are different between pre-vmr and 8.0+? You may need to specify them manually here or amend this script.